manifests/prometheus-adapter-clusterRoleAggregatedMetricsReader.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  labels:
+    rbac.authorization.k8s.io/aggregate-to-admin: "true"
+    rbac.authorization.k8s.io/aggregate-to-edit: "true"
+    rbac.authorization.k8s.io/aggregate-to-view: "true"
+  name: system:aggregated-metrics-reader
+rules:
+- apiGroups:
+  - metrics.k8s.io
+  resources:
+  - pods
+  verbs:
+  - get
+  - list
+  - watch
```

Comment:
watch is not technically available for this API

Reply:
This is technically true :+1: I am fine with removing it, leaving this here will make it readily available for the potential inclusion which is discussed upstream.

Reply:
Agreed. Doesn't hurt. lgtm